packages/abi-typegen/package.json

```diff
@@ -43,9 +43,10 @@
     "dist"
   ],
   "scripts": {
-    "pretest": "pnpm build:forc",
+    "pretest": "run-s build:forc-callpaths build:forc",
     "build": "tsup",
     "build:forc": "pnpm fuels-forc build -p test/fixtures/forc-projects --release",
+    "build:forc-callpaths": "pnpm fuels-forc build -p test/fixtures/forc-projects --json-abi-with-callpaths",
```

Comment:
I'm using the fact that this generates outputs in the `debug` folder so that I can run [this](https://github.com/FuelLabs/fuels-ts/pull/2372/files#diff-7e78de98857b1c1d9877c6eec968b44cc4e968970a7eccbe1e64cc04ad381e53R17) test on both versions of the abi, expecting the same output in both cases.